tests-e2e/cypress/e2e/edx-frontend-district/accessDistrictUsers.cy.ts

```diff
@@ -168,19 +168,12 @@ describe('Access District Users Page Tests', () => {
       let tempLastName2 = '';
 
       before(() => {
-        cy.task<DistrictUserOptions, EdxUserEntity>('setup-districtUser', {
-          digitalId: crypto.randomUUID(), districtRoles: ['EDX_DISTRICT_ADMIN'], districtCodes: ['998']
-        }).then((user: EdxUserEntity) => {
-          tempUserId = user.edxUserID;
-          tempFirstName = user.firstName;
+        cy.task<DistrictUserOptions, EdxUserEntity>('setup-districtUser', { digitalId: crypto.randomUUID(), districtRoles: ['EDX_DISTRICT_ADMIN'], districtCodes: ['998']
```

Comment:
Up until this change was made the file was kept within a tidy 120 columns.  This may seem like a nitpick, but I would prefer to keep it that way.